scripts/dcou-tainted-packages.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# shellcheck disable=SC2034 # This file is intended to be `source`d
+declare dcou_tainted_packages=(
+  solana-accounts-bench
+  solana-banking-bench
+  agave-ledger-tool
+  solana-bench-tps
+  agave-store-tool
+  agave-store-histogram
+  agave-accounts-hash-cache-tool
+  solana-dos
```

Comment:
`solana-dos` is need to be tainted as well... well, `scripts/check-dev-context-only-utils.sh tree` has a corner case bug... I'm planning to fix it later.